docs/_posts/testing_a_tricky_edge_case_in_a_tuf_client.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Testing a tricky edge case in a TUF client"
+author: Ivana Atanasova
+---
+
+Usually the TUF Specification creates an impression of simple and straightforward approach to address software update systems security gaps. In the next few paragraphs we'll try to convince you that the devil is in the details.
```

Comment:
I am intrigued in hearing from the rest of the team if they think the specification is simple and straightforward. :D
Maybe it's not bad in using a little advertainment language.

Reply:
The key is in "creates an impression" :D